main.tf

```diff
@@ -2709,15 +2709,69 @@ module "vpa" {
   tags = var.tags
 }
 
-#-----------------Kubernetes Add-ons----------------------
 
-module "fargate_fluentbit" {
-  count         = var.enable_fargate_fluentbit ? 1 : 0
-  source        = "./modules/fargate-fluentbit"
-  addon_config  = var.fargate_fluentbit_addon_config
-  addon_context = local.addon_context
+################################################################################
+# Fargate Fluentbit
+################################################################################
+# Help on Fargate Logging with Fluentbit and CloudWatch
+# https://docs.aws.amazon.com/eks/latest/userguide/fargate-logging.html
+resource "kubernetes_namespace_v1" "aws_observability" {
+  count = var.enable_fargate_fluentbit ? 1 : 0
```

Comment:
I would check through past issues to see if anyone has raised a request for using an existing namespace, or externally created namespace. Not a blocker since we have the conditional so we can add an additional variable in the future, but if its been requested before we should add it now